packages/failurechecker/package.json

```diff
@@ -28,11 +28,10 @@
     "lint": "gts check"
   },
   "dependencies": {
-    "@octokit/rest": "^18.0.6",
     "gcf-utils": "^6.2.0"
   },
   "devDependencies": {
-    "@octokit/types": "^5.5.0",
+    "@octokit/types": "^6.0.0",
```

Comment:
Elsewhere this was replaced with `@octokit/openapi-types` - I'm curious - why keep this package here?  Is it still needed?

Reply:
I think `@octokit/types` can be removed altogether, it doesn't seem to be used at all